.appveyor.yml

```
@@ -23,7 +23,7 @@ install:
   - pyinstaller --name oj --onefile onlinejudge\_implementation\main.py
 
 test_script:
-  - python setup.py test
+  - pytest
```

```suggestion
  - pytest tests/*.py
```

ファイルを指定しないと `**/*test*.py` という名前のファイル (`onlinejudge/_implementation/command/test_reactive.py` とか) しか対象にならないぽい